test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -34,13 +37,19 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, 1, Integer.valueOf(1));
         assertNotUnique(null, null);
         assertNotUnique(null, "a", "b", null);
+        assertIsAnyNull(null, 1, "a", null);
```

Comment:
These test cases only test for `assertFalse`, you are highly recommended to test for `assertTrue` as well